Sources/swift-format/Subcommands/LintFormatOptions.swift

```diff
@@ -26,6 +26,17 @@ struct LintFormatOptions: ParsableArguments {
       """)
   var configuration: String?
 
+  /// A list of comma-separated "start:end" pairs specifying UTF-8 offsets of the ranges to format.
+  ///
+  /// If not specified, the whole file will be formatted.
+  @Option(
+    name: .long,
+    help: """
+      A list of comma-separated "start:end" pairs specifying UTF-8 offsets of the ranges to format.
```

Comment:
Instead of having a comma-separated list, wouldn’t it be nicer if you could specify `--range` or something multiple times to format multiple ranges? Seems like a more idiomatic CLI to me.

Reply:
I _think_ for array-typed `@Option`s, the [default behavior](https://swiftpackageindex.com/apple/swift-argument-parser/1.4.0/documentation/argumentparser/option/init(name:parsing:help:completion:)-238hg) should already be to append multiple occurrences of the flag into an array.

But since this property is actually a `[Range<Int>]?`, the optional bit might be interfering with the API. Can we drop the optional? We shouldn't need to distinguish between `nil` and an empty array here.

Reply:
This also means the `ExpressibleByArgument` part below should probably be updated so that just `Range<Int>` conforms to it, instead of `[Range<Int>]`. Then if we make this property non-optional `[Range<Int>]`, everything else will fall out from there.

Reply:
Ah. I hadn't thought of the optional bit being what was getting in the way....

Reply:
And it works fine. So simple.

Reply:
Gah! (Now that I pushed the change.) No, it doesn't work fine. This cause _at least one_ `--offsets` argument to be required, which we definitely don't want. (This is what I was hitting when I went down this road before.) But this is the main remaining issue on the PR, so I'll see if I can figure out a solution....

Reply:
You might need to assign the property an explicit default value: `var offsets: [Range<Int>] = []`

Reply:
Thanks. Man, I hate programming in C++. Oh... wait.... 🙃